iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ValidationUtil.java

```diff
@@ -357,13 +465,40 @@ public static void checkRatioTapChangerRegulation(Validable validable, boolean r
     }
 
     public static void checkRatioTapChangerRegulation(Validable validable, boolean regulating,
-                                                      Terminal regulationTerminal, double targetV, Network network) {
+        Terminal regulationTerminal, double targetV, double targetDeadband, Network network) {
+        checkRatioTapChangerRegulation(validable, regulating, true, regulationTerminal, targetV, targetDeadband, network);
+    }
+
+    /**
+     * @deprecated
+     * Use {@link #checkRatioTapChangerRegulation(Validable, boolean, Terminal, double, double, Network)} instead
+     */
+    @Deprecated(since = "4.4.0")
+    public static void checkRatioTapChangerRegulation(Validable validable, boolean regulating,
+        Terminal regulationTerminal, double targetV, Network network) {
         checkRatioTapChangerRegulation(validable, regulating, true, regulationTerminal, targetV, network);
     }
 
+    /**
+     * For validating ratio tapChanger controls.
+     * If the control is enabled voltage setpoint, target deadband and regulating terminal must be defined and valid.
+     */
+    public static void checkRatioTapChangerRegulation(Validable validable, boolean regulating, boolean loadTapChangingCapabilities,
+        Terminal regulationTerminal, double targetV, double targetDeadband, Network network) {
+        if (!loadTapChangingCapabilities) {
+            return;
```

Comment:
Should not return if `!loadTapChangingCapabilities`, should log warnings (see old method)

Reply:
Done. When `!loadTapChangingCapabilities` a warning is logged.